.github/workflows/mega-linter.yml

```
@@ -41,133 +122,231 @@ jobs:
     name: MegaLinter
     runs-on: ubuntu-latest
 
-    # Give the default GITHUB_TOKEN write permission to commit and push, comment
-    # issues & post new PR; remove the ones you do not need
+    ##
+    # This job's `GITHUB_TOKEN` or your added `PAT` must have the following
+    # permissions enabled to `write` for MegaLinter to function.
+    #
+    # Always aim to provide as few permissions as possible for personal access
+    # tokens. By default, all permissions are set to read only. Write
```

```suggestion
    # tokens. By default, all permissions are set to read-only. Write
```
To match mega-linter-runner/generators/mega-linter/templates/mega-linter.yml